04-files.md

```diff
@@ -16,8 +16,8 @@ The only thing that's missing is a library with a rather unpleasant name:
 import glob
 ~~~
 
-The `glob` library contains a single function, also called `glob`,
-that finds files whose names match a pattern.
+The `glob` library contains a function, also called `glob`,
+that finds files and directories whose names match a pattern.
```

Comment:
`glob.glob()` **does** return both files and directories, not sure what you are asking me to clarify here.